pkgs/stdenv/generic/default.nix

```diff
@@ -41,7 +42,7 @@ let lib = import ../../../lib; in lib.makeOverridable (
 }:
 
 let
-  defaultNativeBuildInputs = extraBuildInputs ++
```

Comment:
N.B. It's because of this mismatch that I blanket-rename the arguments in the stdenvs other than Darwin's.